src/styl/_themes/_skin.styl

```diff
@@ -11,7 +11,7 @@
 // .skin-chat        - Text skin: "chat"
 // .skin-exclusive   - Text skin: "exclusif", text color: highlight
 // .skin-fakeoff     - Text skin: "fake off", text color: default
-// .skin-lastminute  - Text skin: "dernière minute", text color: white, background-color: highlight
+// .skin-lastminute  - Text skin: "alerte info", text color: white, background-color: highlight
```

Comment:
I'm not a fan to keep the classname `lastminute ` and change the wording.. What do you think @ryuran ?

Reply:
I was asking the same question. But in seems that in database lastminute skin is saved. 
It means old article will lose the property if not resave

Reply:
I'm not saying to necessary overwrite this skin but create another one. And if "old" article needs this skin, it'll have to be done manually.. Because, what you are doing here is basically adding a new skin

Reply:
As we said in poker, we can overwrite the current skin. Lastminute skinned articles are only available on the homepage, plus it won't obviously never ever be a lastminute article again.

Reply:
Ok with @nredoulez and @JamesLaviron the classename should be renamed